### YamlMime:TSEnum
name: CustomFunctions.ErrorCode
uid: custom-functions-runtime!CustomFunctions.ErrorCode:enum
package: custom-functions-runtime!
fullName: CustomFunctions.ErrorCode
summary: >-
  Codes d’erreur pour les fonctions personnalisées. Les codes d’erreur s’affichent dans la cellule qui a appelé la fonction.


  Les messages d’erreur personnalisés s’affichent en plus de ces codes d’erreur. Les messages personnalisés s’affichent dans le menu indicateur d’erreur, accessible en plaçant le curseur sur l’indicateur d’erreur sur chaque cellule avec une erreur.
remarks: ''
isPreview: false
isDeprecated: false
fields:
- name: divisionByZero
  uid: custom-functions-runtime!CustomFunctions.ErrorCode.divisionByZero:member
  package: custom-functions-runtime!
  summary: 'Ce code d’erreur indique que la fonction utilisée est divisée par zéro ou des cellules vides. Un message d’erreur personnalisé ne peut pas être utilisé. \[[Ensemble d’API : CustomFunctionsRuntime 1,3](/office/dev/add-ins/excel/custom-functions-requirement-sets)\]'
  value: "\"#DIV/0!\""
- name: invalidName
  uid: custom-functions-runtime!CustomFunctions.ErrorCode.invalidName:member
  package: custom-functions-runtime!
  summary: 'Ce code d’erreur indique qu’il existe une faute de frappe dans le nom de la fonction. Notez que ce code d’erreur est pris en charge en tant qu’erreur de saisie d’une fonction personnalisée, mais pas en tant qu’erreur de sortie de fonction personnalisée. Un message d’erreur personnalisé ne peut pas être utilisé. \[[Ensemble d’API : CustomFunctionsRuntime 1,3](/office/dev/add-ins/excel/custom-functions-requirement-sets)\]'
  value: "\"#NAME?\""
- name: invalidNumber
  uid: custom-functions-runtime!CustomFunctions.ErrorCode.invalidNumber:member
  package: custom-functions-runtime!
  summary: 'Ce code d’erreur indique qu’il y a un problème avec un numéro dans la fonction. Un message d’erreur personnalisé ne peut pas être utilisé. \[[Ensemble d’API : CustomFunctionsRuntime 1,3](/office/dev/add-ins/excel/custom-functions-requirement-sets)\]'
  value: "\"#NUM!\""
- name: invalidReference
  uid: custom-functions-runtime!CustomFunctions.ErrorCode.invalidReference:member
  package: custom-functions-runtime!
  summary: 'Ce code d’erreur indique que la fonction fait référence à une cellule non valide. Notez que ce code d’erreur est pris en charge en tant qu’erreur de saisie d’une fonction personnalisée, mais pas en tant qu’erreur de sortie de fonction personnalisée. Un message d’erreur personnalisé ne peut pas être utilisé. \[[Ensemble d’API : CustomFunctionsRuntime 1,3](/office/dev/add-ins/excel/custom-functions-requirement-sets)\]'
  value: "\"#REF!\""
- name: invalidValue
  uid: custom-functions-runtime!CustomFunctions.ErrorCode.invalidValue:member
  package: custom-functions-runtime!
  summary: 'Ce code d’erreur indique qu’une valeur dans la fonction est d’un type de données incorrect. Un message d’erreur personnalisé peut être utilisé en plus du code d’erreur, si vous le souhaitez. \[[Ensemble d’API : CustomFunctionsRuntime 1,3](/office/dev/add-ins/excel/custom-functions-requirement-sets)\]'
  value: "\"#VALUE!\""
- name: notAvailable
  uid: custom-functions-runtime!CustomFunctions.ErrorCode.notAvailable:member
  package: custom-functions-runtime!
  summary: 'Ce code d’erreur indique que la fonction ou le service n’est pas disponible. Un message d’erreur personnalisé peut être utilisé en plus du code d’erreur, si vous le souhaitez. \[[Ensemble d’API : CustomFunctionsRuntime 1,3](/office/dev/add-ins/excel/custom-functions-requirement-sets)\]'
  value: "\"#N/A\""
- name: nullReference
  uid: custom-functions-runtime!CustomFunctions.ErrorCode.nullReference:member
  package: custom-functions-runtime!
  summary: 'Ce code d’erreur indique que les plages de la fonction ne se croisent pas. Un message d’erreur personnalisé ne peut pas être utilisé. \[[Ensemble d’API : CustomFunctionsRuntime 1,3](/office/dev/add-ins/excel/custom-functions-requirement-sets)\]'
  value: "\"#NULL!\""
metadata:
  ms.openlocfilehash: cec80f674bc573e3e3d3679ab96792ea5e469ce3
  ms.sourcegitcommit: a6ecbc9abca2ab5c82af4c869e3e2d72c999a4c1
  ms.translationtype: MT
  ms.contentlocale: fr-FR
  ms.lasthandoff: 12/02/2020
  ms.locfileid: "49547340"
